### PR TITLE
cleanup redundant text in modules utilizing mkEnableOption

### PR DESCRIPTION
**Pull request by aanderse**
###### Motivation for this change
Most of these looks horrible when searching through [options](https://nixos.org/nixos/options.html).

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: nixos/modules/services/hardware/bluetooth.nix [hunk 1]
```diff
@@ -13,7 +13,7 @@ in {
   options = {
 
     hardware.bluetooth = {
-      enable = mkEnableOption "support for Bluetooth.";
+      enable = mkEnableOption "support for Bluetooth";
```
Comment (teto) [inline nixos/modules/services/hardware/bluetooth.nix]: Capitalize first letter ?

File: nixos/modules/services/mail/offlineimap.nix [hunk 2]
```diff
@@ -7,7 +7,7 @@ let
 in {
 
   options.services.offlineimap = {
-    enable = mkEnableOption "Offlineimap, a software to dispose your mailbox(es) as a local Maildir(s).";
+    enable = mkEnableOption "Offlineimap, a software to dispose your mailbox(es) as a local Maildir(s)";
```
Comment (rycee) [inline nixos/modules/services/mail/offlineimap.nix]: The [official site](https://www.offlineimap.org/) seems to consistently use OfflineIMAP.
```suggestion
    enable = mkEnableOption "OfflineIMAP, a software to dispose your mailbox(es) as a local Maildir(s)";
```

**Review by rycee — COMMENTED** [review]
Looks good to me.

File: nixos/modules/services/admin/oxidized.nix [hunk 3]
```diff
@@ -7,7 +7,7 @@ let
 in
 {
   options.services.oxidized = {
-    enable = mkEnableOption "the oxidized configuation backup service.";
+    enable = mkEnableOption "the oxidized configuation backup service";
```
Comment (rycee) [inline nixos/modules/services/admin/oxidized.nix]: Just noticed this:
```suggestion
    enable = mkEnableOption "the oxidized configuration backup service";
```

**Comment by rycee** [comment]
Thanks! Squashed and rebased to master in 4a11ce7f261396d06a92b76df702b16dc01b81ca.

**Comment by aanderse** [comment]
Thanks for your careful review @rycee!

**Pull request closed without merging**